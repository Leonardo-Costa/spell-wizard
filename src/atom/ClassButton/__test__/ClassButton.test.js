import React from 'react';
import renderer from 'react-test-renderer';

import {ClassButton} from '../index';

it('renders correctly', () => {
    const tree = renderer.create(<ClassButton />).toJSON();
    expect(tree).toMatchSnapshot();
  });