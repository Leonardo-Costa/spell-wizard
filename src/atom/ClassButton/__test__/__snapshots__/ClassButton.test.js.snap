// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessible={true}
  collapsable={false}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "backgroundColor": "#202020",
      "borderRadius": 50,
      "marginBottom": 10,
      "opacity": 1,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#FFFFFF",
        "fontSize": 18,
        "fontWeight": "500",
        "paddingHorizontal": 15,
        "paddingVertical": 10,
        "textTransform": "capitalize",
      }
    }
  />
</View>
`;
